browser/css/jsdialogs.css

```diff
@@ -1899,3 +1899,13 @@ kbd,
 .jsdialog-container [id^='info-modal-label'] {
 	display: flex;
 }
+
+/* Custom props */
+#customprops .jsdialog .ui-grid .menubutton {
```

Comment:
customprops is quite generic id, which can appear somewhere else.
Please add one more id which is specific for that dialog:
`#TheMainDialogId #customprops .jsdialog .ui-grid .menubutton`

We always add for every dialog such Id.